src/databricks/labs/ucx/hive_metastore/table_migration_status.py

```diff
@@ -80,8 +80,8 @@ def __init__(self, ws: WorkspaceClient, sbe: SqlBackend, schema, table_crawler:
         self._ws = ws
         self._table_crawler = table_crawler
 
-    def index(self) -> TableMigrationIndex:
-        return TableMigrationIndex(list(self.snapshot()))
+    def index(self, *, force_refresh: bool = False) -> TableMigrationIndex:
+        return TableMigrationIndex(list(self.snapshot(force_refresh=force_refresh)))
```

Comment:
I was on the cusp of removing this method: the migration code should just call `snapshot()` directly.

Reply:
Did you do this somewhere else already?

Reply:
@JCZuurmond: No; that area of the code is more complicated than it looks and we have some other priorities right now.

Reply:
oke, let's stick with this solution for now